docs/README.iOS.md

```diff
@@ -75,25 +76,17 @@ git clone https://github.com/xbmc/xbmc kodi
 **[back to top](#table-of-contents)**
 
 ## 4. Configure and build tools and dependencies
-Kodi can be built as either a 32bit or 64bit program. The dependencies are built in `$HOME/kodi/tools/depends` and installed into `/Users/Shared/xbmc-depends`.
+Kodi can be built as a 64bit program for iOS. The dependencies are built in `$HOME/kodi/tools/depends` and installed into `/Users/Shared/xbmc-depends`.
 
 **TIP:** Look for comments starting with `Or ...` and only execute the command(s) you need.
 
 Configure build for 64bit (**recommended**):
```

Comment:
What is the not recommended alternative?

Reply:
Fair point. Removed the recommended.